mkdd_widgets.py

```diff
@@ -37,7 +37,7 @@
 MODE_3D = 1
 
 #colors = [(1.0, 0.0, 0.0), (0.0, 0.5, 0.0), (0.0, 0.0, 1.0), (1.0, 1.0, 0.0)]
-colors = [(0.0,191/255.0,255/255.0), (30/255.0,144/255.0,255/255.0), (0.0,0.0,255/255.0), (0.0,0.0,139/255.0)]
+colors = [(0.0,191/255.0,255/255.0), (30/255.0,144/255.0,255/255.0), (0.0,0.0,255/255.0), (0.0,0.0,139/255.0), (0.0,255/255.0,0.0)]
```

Comment:
Note that `colors` happens to be list of blue-ish colors. I think defining a new variable (e.g. `lap_checkpoint_color`) would make more sense.